content/doc/book/pipeline/scaling-pipeline.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: section
+title: Scaling Pipelines
+---
+:description:
+:author: Sam Van Oort
+:sectanchors:
+:toc:
+:hide-uri-scheme:
+
+
+= Using Speed/Durability Settings To Reduce Disk I/O Needs
+
+One of the main bottlenecks in Pipeline is that it writes transient data to disk *FREQUENTLY* in order to be able to resume Pipelines when Jenkins restarts or if something goes wrong and it crashes.  This durability is useful for many users but its performance cost can be a problem. 
+
+Pipeline now includes features to let users improve performance by reducing how much data is written to disk and how often it is written -- at a small cost to durability.  In some special cases, users may not be able to resume or visualize running Pipelines if Jenkins shuts down suddenly without getting a chance to write data. 
+
+Because these settings include a trade-off they are initially opt-in. Users need to explicity set a _Speed/Durability Setting_ for Pipelines.  Initially they will default to the least aggressive setting.  Note that the same plugin updates including these new settings also include additional I/O optimizations. 
```

Comment:
A couple things here.

1. Potential confusion could wrise from the first sentence. If the settings are initially opt-in, why do users need to explicitly set a Speed/Durability setting?

2. "...least aggressive" could be ambiguous. I know you mean "least aggressive in terms of performance optimizations," but others might not. Could just as easily be "least aggressive in terms of durability."

Possible reword:

"Because these new performance settings introduce a trade-off of speed versus durability, they are initially opt-in. To enable the performance-optimized modes, users need to explicitly set a _Speed/Durability Setting_ for Pipelines. If no explicit choice is made, Pipelines will remain in the default setting of MAX_DURABILITY, and write to disk frequently as they have in the past.